packages/@atjson/document/src/annotation.ts

```diff
@@ -142,7 +142,7 @@ export abstract class Annotation<Attributes = {}> {
   }
 
   readonly type: string;
-  abstract rank: number;
+  abstract get rank(): number;
```

Comment:
For anyone who's interested, this needs to be defined as a class method as the babel transform-class-properties plugin turns class properties into instance properties set in the constructor, which means they are not in the prototype and can't be called via things like `super.rank`

Reply:
this is.... so unfortunate